src/playercards/cards/ScrollofSecrets.ttslua

```diff
@@ -0,0 +1,92 @@
+-- this script is shared between the lvl 0 and lvl 3 versions of Scroll of Secrets
+local mythosAreaApi = require("core/MythosAreaApi")
+local playmatApi    = require("playermat/PlaymatApi")
+
+-- get class via metadata and create context menu accordingly
+function onLoad()
+  local notes = JSON.decode(self.getGMNotes())
+  if notes then
+    createContextMenu(notes.class)
+  else
+    print("Missing metadata for Scroll of Secrets!")
+  end
+end
+
+function createContextMenu(class)
+  local colorList = Player.getAvailableColors()
+
+  if class == "Seeker|Mystic" then
```

Comment:
This should probably be checking IDs rather than classes

Reply:
Fair point. Updated